include/mgmt/mcumgr/smp_shell.h

```diff
@@ -26,17 +26,19 @@ struct smp_shell_data {
 };
 
 /**
- * @brief Attempts to process a received byte as part of an SMP frame.
+ * @brief Attempt to process received bytes as part of an SMP frame.
  *
- * This function should be called with every received byte.
+ * Called to scan through buffer and consume all bytes that constitute SMP
+ * frame.
```

Comment:
This is not entirely true. If there will be SMP frame on the beginnig of buffer, then shell bytes in the middle, SMP frame once again at the end of buffer, then only SMP frame from beginning of buffer will be consumed.

Reply:
True, will fix comment.

Reply:
Fixed.